CMakeLists.txt

```diff
@@ -9,6 +9,12 @@ project(whole-body-estimators LANGUAGES CXX
 set(CMAKE_CXX_STANDARD 14)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
+## avoid having M_PI error by importing math symbols from standard cmath
+add_definitions(-D_USE_MATH_DEFINES)
```

Comment:
Only for me to understand, could you point me to a line where there's a potential math symbol error?

Reply:
If you search for "error" in this [raw log](https://pipelines.actions.githubusercontent.com/YOZBtIxyFZC86xEU2CiDMdyRfOcLH42OneCPmug013AruDqGoK/_apis/pipelines/1/runs/6/signedlogcontent/4?urlExpires=2020-03-16T13%3A55%3A33.1498881Z&urlSigningMethod=HMACV1&urlSignature=aTIwk8On6sVGKYfIRGP4TtZcWL1sDHFE2169iL7y5Do%3D) of that specific job in the workflow, you will find the problem. The raw log can be accessed by pressing `...` in this page https://github.com/prashanthr05/whole-body-estimators/runs/506436067?check_suite_focus=true

This was actually reported by @traversaro  in https://github.com/robotology/whole-body-estimators/issues/6#issuecomment-599040119

Reply:
Clicking the raw log points to an expired url. It's better to go directly to the actions tab in the repository and check the status.